code/modules/ninja/suit/suit.dm

```diff
@@ -71,15 +71,22 @@ Contents:
 
 	//Cell Init
 	cell = new/obj/item/stock_parts/cell/high
-	cell.charge = 60000 // larger as it now heats
-	cell.maxcharge = 60000
+	cell.charge = 9000 // larger as it now heats
+	cell.maxcharge = 9000
```

Comment:
The maxcharge of this cell was originally 10,000, not 9,000. In other words, the cell should start at 90% charge, if we're reverting it back to how it used to be.

You can still keep it this way if you want to, but keep in mind that, if you make ninjas start with a capacity 9,000 cell, they can upgrade straight to a NORMAL yellow slime core instead of needing to wait for a hypercharged one. If that's fine with you, then go ahead; I won't stop you, I'm just making sure you're aware of the consequences of what you're doing.

Reply:
I did not realize that, great catch, I have removed the max charge on this.